pom.xml

```diff
@@ -66,6 +66,11 @@
             <type>war</type>
             <scope>runtime</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apereo.cas</groupId>
```

Comment:
Cool. My vote would be to take up that route; the location of the downloaded jar can be anywhere (possibly in the target directory or the build directory if this was a gradle build). However, it would be nice to auto-calculate and auto-extract the CAS version that is defined. With Gradle, it's easy to do that with top and grep commands, but with Maven you may need to resort to other means (possible to externalize the versions into a .properties file?)